ocaml/idl/markdown_backend.ml

```diff
@@ -130,7 +130,7 @@ let markdown_section_of_message printer x =
   printer (sprintf "%s (%s)"
     x.msg_name
     (String.concat ", "
-      ((if x.msg_session then "session_id s" else "")::
+      ((if x.msg_session then ["session ref session_id"] else []) @
```

Comment:
Another thing I was considering here was to drop the param names here and have only the types, for brevity. The name are already in the tables immediately below the function prototype. This is how the online docs are as well.